conda/dev.yaml

```diff
@@ -6,12 +6,8 @@ dependencies:
   - docker-compose
   - libblas
   - nodejs
-  - pandas
   - pip
   - poetry >= 1.3.2
   - psycopg2
   - python 3.9.*
   - shellcheck
-  - sqlalchemy
-  - pip:
-    - pysus
```

Comment:
thanks for removing these duplicated deps
but you need to add pysus as a dep using poetry